ouroboros-network/src/Ouroboros/Network/Mux.hs

```diff
@@ -35,30 +37,38 @@ import           Ouroboros.Network.Channel
 --
 data OuroborosApplication (appType :: AppType) peerid ptcl m bytes a b where
      OuroborosInitiatorApplication
-       :: (peerid -> ptcl -> Channel m bytes -> m a)
+       :: ((ptcl -> MiniProtocolInitiatorControl m a) -> m ())
```

Comment:
Please expand haddock documentation of this type to include the role and purpose of this field.

Reply:
Will do.